js/siteorigin-panels/view/builder.js

```diff
@@ -579,7 +579,9 @@ module.exports = Backbone.View.extend( {
 
                     // Trigger a focusout (mainly for Yoast SEO)
                     $('#content').focusout();
```

Comment:
Your original PR had some lines removed here. That no longer necessary?

https://github.com/siteorigin/siteorigin-panels/pull/143/files#diff-83dd5f80393228689a7367e94915ca16L1983